doc/operator-user-guide.md

```diff
@@ -711,6 +713,61 @@ spec:
 Only when the underlying PersistentVolume's storageclass allows resizing, storage resource requirements can be modified after installation.
 Check [Expanding persistent volumes](https://docs.openshift.com/container-platform/4.5/storage/expanding-persistent-volumes.html) official doc for more information.
 
+#### Setting custom PriorityClassName
+PriorityClassName specifies the Pod priority.  
```

Comment:
Please, provide link for priority class name doc. Maybe like [this](https://docs.openshift.com/container-platform/4.13/nodes/pods/nodes-pods-priority.html)?